common/source/docs/common-generators.rst

```diff
@@ -0,0 +1,23 @@
+.. _common-generators:
+
+==========
+Generators
+==========
+
+.. figure:: ../../../images/generators.png
+   :target: ../_images/generators.png
+
+Support for generators is currently available in master.  Within ArduPilot the term generator is used to broadly group all devices that convert some form or fuel into electrical power.  Hence, the list of currently supported generators includes hydrogen fuel cells.
```

Comment:
"or" should be "of" I think.

No big deal but it is good to be concise so we could drop, "Hence, the list of currently supported generators includes hydrogen fuel cells."

I'd probably replace, "Support for generators is currently available in master" with "ArduPilot 4.1 (and higher) support generators which can be used to extend the flight time".

Reply:
Fixed.